README.md

```diff
@@ -1,78 +1,405 @@
+firstName,age,nationalInsurance
+-619248029,71,"HT849919"
+08-12-2001 02:53:16,15,
+"Lorem Ipsum",11,
+"Lorem Ipsum",71,"WX004081"
+1263483797,19,"HG054666"
+,75,"GE023082"
+"Lorem Ipsum",59,"ZM850737C"
+[...]
+~~~
 
-* [Generator](./generator/README.md)
-  * [Generation algorithm](./generator/docs/GenerationAlgorithm.md)
-  * [Set data generation process](./generator/docs/SetRestrictionAndGeneration.md)
-  * [Decision trees](./docs/DecisionTrees/DecisionTrees.md)
+However, it might be a surprise to see nulls, numbers and dates as values for the `firstName` field alongside strings that do not match the regex given in the profile. This is because these are all defined as a single rule within the profile. You have a couple of options if you want to ensure that `firstName` is not null and a string, the first is to inform the generator that it should not violate specific constraint types:
 
-* [Frequently asked questions](docs/FrequentlyAskedQuestions.md)
+~~~
+$ java -jar generator.jar generate --violate --dont-violate=ofType null \
+  --max-rows=100 --overwrite profile.json out
+~~~
 
-## Contributing
+Or, alternatively, you can re-arrange your constraints so that the ones that define types / null, are grouped as a single rule. After By re-grouping constraints, the following output, with random strings that violate the regex constraint, is generated:
 
-See [contributing](./.github/CONTRIBUTING.md) guidance and the [key decisions log](docs/KeyDecisions.md).
+```
+firstName,age,nationalInsurance
+"�",43,"PT530853D"
+"뷇",56,"GE797875M"
+"邦爃",84,"JA172890M"
+"J㠃懇圑㊡俫杈",32,"AE613401F"
+"俵튡",38,"TS256211F"
+"M",60,"GE987171M"
+"M",7,
+"Mꞎኅ剭Ꙥ哌톞곒",97,"EN082475C"
+")",80,"BX025130C"
+",⑁쪝",60,"RW177969"
+"5ᢃ풞ﺯ䒿囻",57,"RY904705" 
+[...]
+```
 
-### Development setup
+## Next steps
 
-This product requires **Java version 1.8** to be installed. Later versions are not supported at present.
-* Follow the steps in the [setup instructions](./generator/docs/GeneratorSetup.md) then continue with these steps.
-* Build the solution using your preferred method - the project is fully compatible with IntelliJ and Eclipse. 
-You can use Maven if you prefer by executing the following command in the project root directory:
-    ```bash
-    mvn install
-    ```
-* Run the generator - see the [examples here](./generator/README.md).
-* To generate a project documentation website run the 
-[maven site](https://maven.apache.org/guides/mini/guide-site.html) command. 
-The website will contain information about the projects such as libraries used and unit test report.
-Once generated the website can be found at `target\site\index.html` in each of the project directories.
-    ```bash
-    mvn site
-    ```
+That's the end of our getting started guide. Hopefully it has given you a good understanding of what the DataHelix generator is capable of. If you'd like to find out more about the various constraints the tool supports, the [Profile Developer Guide](???) is a good next step.
```

Comment:
Might want to highlight the [examples folder](https://github.com/ScottLogic/datahelix/tree/master/examples) here as well as any documentation.